bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
+    while f != os.path.sep:
+        if os.path.ismount(f): return f
+        f = os.path.realpath(os.path.join(f, os.pardir))
+    return f
+
+
+def get_file_system_type(filepath):
+    """
+    Return the file system type the file object file resides on.
+    """
+
+    # normalize,resolve and get absolute path to the file object
+    f = os.path.realpath(filepath)
+    proc = subprocess.Popen(['df', '-T', '--', f], stdout=PIPE, stderr=PIPE)
+    stdout, stderr = proc.communicate()
+    ftype = None
+    if stdout:
+        t = stdout.splitlines()
```

Comment:
't' is a bad name. Could you please improve it? E.g., use 'entries', or 'split_line', or something else that is logical and at least somewhat self-documenting. Also, it would be nice to add a small comment with an example of the output string that you are parsing here, makes the code easier to read. Right now I have no clue what is going on and I have to run that 'df -T' and check how the output looks like. A comment would help.